python/remage/ipc.py

```diff
@@ -11,7 +11,28 @@
 
 
 def handle_ipc_message(msg: bytes) -> tuple[bool, list, bool]:
```

Comment:
```suggestion
def handle_ipc_message(msg: str) -> tuple[bool, list, bool]:
```

yeah, actually it already was converted to str below...